apps/docs/package.json

```
@@ -8,11 +8,13 @@
     "clear": "docusaurus clear",
     "write-translations": "docusaurus write-translations",
     "write-heading-ids": "docusaurus write-heading-ids",
-    "typecheck": "tsc"
+    "typecheck": "tsc",
+    "build": "docusaurus build"
```

issue: We don't need this script since we have `nx build docs`
```suggestion
    "typecheck": "tsc"
```